config/playground/helm/dev/helm-opensearch-dashboards.yaml

```diff
@@ -60,24 +61,6 @@ securityContext:
   runAsUser: 1000
 
 config:
-  # Default OpenSearch Dashboards configuration from docker image of Dashboards
```

Comment:
let's not remove config value here, and use this config to control the feature flag, we don't need to modify the inside of yml file of docker image, we just need to install plugins and add the required the value here.

Reply:
Yeah, the reason I put into the extra config folder is to build it into image - but I hear you that it would be inconvenient to change configs without building new images. I will update the set up to put it back for now. In the long run, I still think we should push for making changing in the helm chart to make it more extensible - https://github.com/opensearch-project/helm-charts/issues/341.

Reply:
Helm chart have function here is to override or replace the opensearch_dashboards.yaml, which give cx ability to control inside of docker image's opensearch_dashboards.yaml. we should do the same way. you can double check if not use your configure value compare function, it still can do the work, it will override or replace whole configure to inside of docker image config

Reply:
Yeah, discussed with @Flyingliuhub offline, and will have config back to the chart values yaml and remove from the extra config settings for playground. The build custom image to install plugin and add additional config can be its own action in the future and be available for other users to use - at that time, we can optimize it so that users can add additional config together(possibly through specifying a storage path for the config or use command to write into the file) when installing plugins into the image.

Reply:
Made the following change in [new commit](https://github.com/opensearch-project/dashboards-anywhere/pull/91/commits/f040d023582c47d0b74cce5d1a3c890f55d171df) based on discussion:
- Moved config back to chart's value yaml 
- removed config from extra config folder
- changed to set the tracking id when installing the chart and when tracking id is provided